samples/MvvmSampleUwp/Helpers/TitleBarHelper.cs

```diff
@@ -28,11 +28,11 @@ public static void StyleTitleBar()
         titleBar.ButtonInactiveBackgroundColor = Colors.Transparent;
 
         // Theme aware colors
-        titleBar.ButtonForegroundColor = titleBar.ButtonHoverForegroundColor = titleBar.ButtonPressedForegroundColor = Colors.White;
-        titleBar.ButtonHoverBackgroundColor = Color.FromArgb(0x20, 0xFF, 0xFF, 0xFF);
-        titleBar.ButtonPressedBackgroundColor = Color.FromArgb(0x40, 0xFF, 0xFF, 0xFF);
-        titleBar.ButtonInactiveForegroundColor = Color.FromArgb(0xC0, 0xFF, 0xFF, 0xFF);
-        titleBar.InactiveForegroundColor = Color.FromArgb(0xA0, 0xA0, 0xA0, 0xA0);
+        //titleBar.ButtonForegroundColor = titleBar.ButtonHoverForegroundColor = titleBar.ButtonPressedForegroundColor = Colors.White;
```

Comment:
Why are these commented out?
If no styling is needed anymore (why?), then these should just be removed.

Reply:
With light+dark theme we don't need those anymore. Removed!